Code2Gether-Discord-Bot.Library/BusinessLogic/ListProjectsLogic.cs

```diff
@@ -51,8 +53,8 @@ private void ListProjects(out string title, out string description)
                 sb.Append(Environment.NewLine);
             }
 
-            title = "List Projects";
+            title = $"List Projects ({projectCount})";
```

Comment:
It doesn't look like you're adding to this variable in the loop.

Reply:
Haha nice catch